che-theia-task-extension/src/node/che-task-runner.ts

```diff
@@ -43,13 +43,16 @@ export class CheTaskRunner implements TaskRunner {
             throw new Error(`Che task config must have 'target.machineName' property specified`);
         }
 
+        const workingDir = taskConfig.target.workingDir;
+
```

Comment:
As I can see we don't use this const in another places, so you can just pass `taskConfig.target.workingDir.` But up to you.